newdevtech/newdev_api
# #166172700 setup eslint

#### What does this PR do?
- This PR sets up eslint for the repository

#### Description of Task to be completed?
- setup eslint
- add .eslintrc file
- add .eslintignore file

#### How should this be manually tested?
- Confirm that you have `eslint` installed in your editor
- Checkout to this branch with `git checkout chore/166172700/setup-eslint`
- Deliberately violate a rule in the `.eslintrc` file. An example would be leaving a `trailing space` or making a line exceed `80` characters.
- Confirm that `Eslint` flags your violation.

#### What are the relevant pivotal tracker stories?
[#166172700](https://www.pivotaltracker.com/story/show/166172700)
